lib/currency-locales/locales/th.yml

```diff
@@ -1,23 +1,47 @@
+---
 th:
   number:
     currency:
       format:
         delimiter: ","
-        format: "%u %n"
+        format: "%n %u"
         precision: 2
         separator: "."
         significant: false
         strip_insignificant_zeros: false
-        unit: "฿"
+        unit: บาท
```

Comment:
Symbol change!

Reply:
Looks like the new options is local script and the previous option is the international option. Judgement call @tmchow

Reply:
```suggestion
        unit: "฿"
```